css/mobile.css

```diff
@@ -11,6 +11,18 @@
     padding-top: 18vh;
   }
   .logo {
-    margin-top: 0;
+    margin-top: 5em;
```

Comment:
I always forget about rem vs em, but I think rem would be more universal last I looked. Check it out.